app/src/main/kotlin/com/wire/android/ui/common/bottomsheet/conversation/HomeSheetContent.kt

```diff
@@ -53,11 +53,13 @@ import com.wire.android.ui.theme.wireTypography
 import com.wire.kalium.logic.data.conversation.MutedConversationStatus
 import com.wire.kalium.logic.data.user.ConnectionState
 
+// items cannot be simplified
+@Suppress("CyclomaticComplexMethod")
 @Composable
 internal fun ConversationMainSheetContent(
     conversationSheetContent: ConversationSheetContent,
 // TODO(profile): enable when implemented
-//    addConversationToFavourites: () -> Unit,
+    changeFavoriteState: (dialogState: GroupDialogState, addToFavorite: Boolean) -> Unit,
```

Comment:
suggestion: move the `// TODO(profile): enable when implemented` line below this one so that it's right next to commented line `// moveConversationToFolder: () -> Unit,`